elytron/src/main/resources/schema/wildfly-elytron_1_0.xsd

```diff
@@ -1,29 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can we please revert the changes made to this file, I can't see what has actually been changed for this PR.

I am not against reformatting in general if the schema needs cleaning up but it should be in it's own PR but it would be better to do it at a time when we don't expect others to be contributing to the schema as well.